sdk/documenttranslation/azure-ai-documenttranslation/tests/preparer.py

```diff
@@ -0,0 +1,49 @@
+
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import os
+import functools
+from devtools_testutils import PowerShellPreparer, AzureMgmtPreparer
+from azure.core.credentials import AzureKeyCredential
+
+
+DocumentTranslationPreparer = functools.partial(
+    PowerShellPreparer,
+    'documenttranslation',
+    documenttranslation_test_endpoint="https://redacted.cognitiveservices.azure.com/",
+    documenttranslation_test_api_key="fakeZmFrZV9hY29jdW50X2tleQ==",
+    documenttranslation_name="redacted",
+    documenttranslation_storage_name="redacted",
+    documenttranslation_storage_key="fakeZmFrZV9hY29jdW50X2tleQ=="
+)
+
+
+class DocumentTranslationClientPreparer(AzureMgmtPreparer):
+    def __init__(self, client_cls, client_kwargs={}, **kwargs):
+        super(DocumentTranslationClientPreparer, self).__init__(
+            name_prefix='',
+            random_name_length=42
+        )
+        self.client_kwargs = client_kwargs
+        self.client_cls = client_cls
+
+    def create_resource(self, name, **kwargs):
```

Comment:
I did something similar for [appconfig here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/appconfiguration/azure-appconfiguration/tests/wrapper.py). This can replace the `if self.is_live` portion by using the values the preparer gives you.

Reply:
smart!